### PR TITLE
Add PodDisruptionBudget for receive hashrings

### PR DESCRIPTION
**Pull request by kakkoyun**
Signed-off-by: Kemal Akkoyun <kakkoyun@gmail.com>

### CODE REVIEW

**Comment by brancz** [comment]
Nice! lgtm :+1:

**Comment by brancz** [comment]
Wait

File: examples/all/manifests/thanos-receive-default-podDisruptionBudget.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: thanos-receive-default
+  namespace: thanos
+spec:
+  maxUnavailable: 0
```
Comment (brancz) [inline examples/all/manifests/thanos-receive-default-podDisruptionBudget.yaml:7]: 0 unavailable? :thinking:

**Comment by brancz** [comment]
Maybe we should only add a PDB if replicas is larger than 1...

**Review by brancz — APPROVED** [review]
nice!

**Comment by bwplotka** [comment]
Thanks!

**Pull request merged**